mozilla/activity-stream
# Bug1505406 - Fix newsletter issues

This patch fixes some issues with defaults and a missing title field in the submit form template. I also added some test data to verify via manual QA.

To test this, please
* ensure `browser.newtabpage.activity-stream.asrouter.devtoolsEnabled` is set to true
* visit about:newtab#asrouter
* select `snippets_local_testing` in the dropdown
* turn on the `NEWSLETTER_TEST_DEFAULTS` snippets by pressing "Show", ensure defaults for the placeholder in the input and button text are showing up
* turn on the `NEWSLETTER_TEST_1` snippet by pressing "Show"
* open network devtools; try submitting an email; ensure the `Params` (body) of the request sent has `en-CA` for the `lang` field
